Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -130,7 +130,7 @@ def audit
       formula_count += 1
       problem_count += fa.problems.size
       problem_lines = format_problem_lines(fa.problems)
-      corrected_problem_count = options[:style_offenses].count(&:corrected?)
+      corrected_problem_count = options[:style_offenses].count(&:corrected?) unless options[:style_offenses].nil?
```

Comment:
`options[:style_offenses]&.count(&:corrected?)` or wrapping in an array would be nicer

Reply:
Dint know this syntax 🙈 Thanks!